packages/eslint-config-kibana/.eslintrc.js

```diff
@@ -116,6 +121,30 @@ module.exports = {
       arrow: true,
     }],
     'react/jsx-first-prop-new-line': ['error', 'multiline-multiprop'],
+    'jsx-a11y/accessible-emoji': 'error',
```

Comment:
__note:__ I have no idea which rules we need, which not, so I'll leave it up to you :) The stricter we're the better for me :)

Reply:
++ we have definitely gone on the stricter route :)